pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -154,7 +155,7 @@ func (p *cadvisorStatsProvider) ListPodStats(_ context.Context) ([]statsapi.PodS
 			podStats.CPU = cpu
 			podStats.Memory = memory
 			podStats.Swap = cadvisorInfoToSwapStats(podInfo)
-			podStats.ProcessStats = cadvisorInfoToProcessStats(podInfo)
```

Comment:
will the cAdvisor still collect it? Should we make it not to collect it?  Would it make sense to merge ProcessStats with the pause process stats?